public/app/features/alerting/unified/home/Insights.tsx

```diff
@@ -97,7 +97,7 @@ const THIS_WEEK_TIME_RANGE = new SceneTimeRange({ from: 'now-1w', to: 'now' });
 
 const namespace = config.bootData.settings.namespace;
```

Comment:
Oh also for consistency, we can access this via `config.namespace` directly (I think that's how it's done in most other places, and definitely how we check it for K8S stuff on the UI)